releases/nightly-build/compose-files/docker-compose-nexus.yml

```diff
@@ -215,8 +215,8 @@ services:
 
   mongo:
     image: nexus3.edgexfoundry.org:10004/docker-edgex-mongo:1.1.0
-    ports:
-      - "27017:27017"
+    expose:
+      - "27017"
```

Comment:
This seems redundant to me- the internal port already exposed on the Dockerfile and you cannot change to expose different port internally. So my suggestion is to remove all of these if we want to disable the port outside so that no mistake can be introduced here.

Reply:
I will remove them all if we decide to have these changes in place. Since we have a no security version, we can always get these ports back if needed.